### PR TITLE
feat(route): add eprice

### PR DESCRIPTION
**Pull request by TonyRL**
## 该 PR 相关 Issue / Involved issue

Close #

## 完整路由地址 / Example for the proposed route(s)

```routes
/eprice
/eprice/hk
/eprice/tw
```

## 新RSS检查列表 / New RSS Script Checklist
  
- [x] New Route
- [x] Documentation
  - [x] CN
  - [ ] EN
- [x] 全文获取 fulltext
  - [x] Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [x] 日期和时间 date and time
  - [x] 可以解析 Parsed
  - [x] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added 
- [ ] `Puppeteer`

## 说明 / Note
Regarding pubDate, upstream provides them in the format of `Tue, 31 Aug 2021 18:29:04 +0800`. No extra timezone adjustment is needed.

### CODE REVIEW

File: docs/new-media.md [hunk 1]
```diff
@@ -224,6 +224,18 @@ pageClass: routes
 
 </Route>
 
+## ePrice
+
+<Route author="TonyRL" example="/eprice/tw" path="/eprice/:region?" :paramsDesc="['地区，预设为tw']">
```
Comment (SettingDust) [inline docs/new-media.md:229]:
```suggestion
<Route author="TonyRL" example="/eprice/tw" path="/eprice/:region?" :paramsDesc="['地区，预设为 tw']">
```

File: lib/routes/eprice/rss.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
+const got = require('@/utils/got');
+const parser = require('@/utils/rss-parser');
+const cheerio = require('cheerio');
+const { parseDate } = require('@/utils/parse-date');
+
+module.exports = async (ctx) => {
+    const region = ctx.params.region || 'tw';
```
Comment (SettingDust) [inline lib/routes/eprice/rss.js:7]:
```suggestion
    const region = ctx.params.region ?? 'tw';
```

File: lib/routes/eprice/rss.js [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+    const feed = await parser.parseURL(`https://www.eprice.com.${region}/news/rss.xml`);
+
+    const items = await Promise.all(
+        feed.items.map(async (item) => {
+            item.link = item.link.replace(/^http:\/\//i, 'https://');
+            const cache = await ctx.cache.get(item.link);
```
Comment (SettingDust) [inline lib/routes/eprice/rss.js:14]: tryGet

File: lib/routes/eprice/rss.js [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+            $('img').each((_, e) => {
+                if ($(e).attr('data-original')) {
+                    $(e).after(`<img src="${$(e).attr('data-original')}" alt="${$(e).attr('alt')}" title="${$(e).attr('title')}">`);
+                    $(e).remove();
+                }
+            });
+
+            const single = {
+                title: item.title,
+                description: $('div.user-comment-block').html() || $('div.content').html(),
```
Comment (SettingDust) [inline lib/routes/eprice/rss.js:50]:
```suggestion
                description: $('div.user-comment-block').html() ?? $('div.content').html(),
```

File: lib/routes/eprice/rss.js [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+            $('div.text-left').remove();
+            $('div.signature').remove();
+
+            // fix lazyload image
+            $('img').each((_, e) => {
+                if ($(e).attr('data-original')) {
+                    $(e).after(`<img src="${$(e).attr('data-original')}" alt="${$(e).attr('alt')}" title="${$(e).attr('title')}">`);
+                    $(e).remove();
+                }
+            });
```
Comment (SettingDust) [inline lib/routes/eprice/rss.js:46]: 替换原有属性即可

**Review by SettingDust — CHANGES_REQUESTED** [review]

File: lib/routes/eprice/rss.js (hunk 5 above)
Comment (TonyRL) [reply lib/routes/eprice/rss.js]: Resolved in https://github.com/DIYgod/RSSHub/pull/8118/commits/7f3ab54d3c029296cc0bacef171f5357533dc2da

File: lib/routes/eprice/rss.js (hunk 3 above)
Comment (TonyRL) [reply lib/routes/eprice/rss.js]: Resolved in https://github.com/DIYgod/RSSHub/pull/8118/commits/551e57edc0f4d9bbec8632fe91a9297e7ec396dc

File: lib/routes/eprice/rss.js [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+                    e = $(e);
+                    if (e.attr('data-original')) {
+                        e.attr('src', e.attr('data-original'));
+                        e.removeAttr('class');
+                        e.removeAttr('data-original');
+                        e.removeAttr('data-title');
+                        e.removeAttr('data-thumbnail');
+                        e.removeAttr('data-url');
+                    }
+                });
```
Comment (SettingDust) [inline lib/routes/eprice/rss.js:54]: 判断属性 filter 更好些，移除属性用白名单遍历移除更好些

**Review by SettingDust — APPROVED** [review]

File: lib/routes/eprice/rss.js (hunk 6 above)
Comment (TonyRL) [reply lib/routes/eprice/rss.js:54]: Your suggested change will have something like
```javascript
$('img').filter(... => ...condition...). // filter loop: O(n)
each((_, e) => { // each loop: O(n)
// attr swap: O(1)
//
// loop through attrs and whitelisting like https://stackoverflow.com/a/3643317
// O(n)
});
```

By considering the time complexity(You should know that there is a really strict function timeout on Vercel and other cloud platforms), the present solution(only one each loop: O(n) ) is better than nested loops (time complexity > O(n) ). 

Someone from [stackoverflow](https://stackoverflow.com/a/5929629) also did profiling and shows that less loop is faster.

**Pull request closed without merging**